src/services/ParticipantProjectService.ts

```diff
@@ -156,17 +156,21 @@ const list = async (
     type LandUse = { landUseModelType: string };
 
     return {
-      cohortName: participant_cohort_name,
-      country: country_name,
+      acceleratorDetails_confirmedReforestableLand,
+      'acceleratorDetails_confirmedReforestableLand(raw)': Number(
```

Comment:
Looks like we no longer need this `(raw)` property. Remove it from the service and other places.
Also remove it from line 59 in this file.
This is all my doing, apologies.